config/routes.rb

```diff
@@ -1,5 +1,14 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  # get '/users', to: 'users#index'
+  # get '/users/:id', to: 'users#show'
+  # get '/users/:author_id/posts', to: 'posts#index'
+  # get '/users/:author_id/posts/:id', to: 'posts#show'
```

Comment:
- [ ] @Nessrine88 Great work adding routes for the users and posts pages, however, you have commented code in this file which is not a good practice kindly remove the commented code and also consider adding the root route.